### PR TITLE
Add english article web video streaming

### PR DESCRIPTION
**Pull request by BernardJeremy**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
50 words might be misspelled, please check them: API, Appshot, bien, Bitmovin, codec, Comparaison, complète, ContentComponent, Dailymotion, de, depuis, d'implémentation, directement, documentée, DRM, d'une, et, Exemple, EXTINF, EXTM, ffmpeg, Fichier, gérer, HDS, hls, HLS, jbernard, jpg, js, JS, lang, le, Librairie, MPD, MPEG, MSS, multi, Nginx, permet, playlist, png, qui, QuickTime, Références, Spécification, très, tv, vidéo, workflow, Youtube

**Comment by ElevenWilson** [comment]
50 words might be misspelled, please check them: API, Appshot, bien, Bitmovin, codec, Comparaison, complète, ContentComponent, Dailymotion, de, depuis, d'implémentation, directement, documentée, DRM, d'une, et, Exemple, EXTINF, EXTM, ffmpeg, Fichier, gérer, HDS, hls, HLS, jbernard, jpg, js, JS, lang, le, Librairie, MPD, MPEG, MSS, multi, Nginx, permet, playlist, png, qui, QuickTime, Références, Spécification, très, tv, vidéo, workflow, Youtube

**Comment by ElevenWilson** [comment]
29 words might be misspelled, please check them: API, Appshot, Bitmovin, codec, ContentComponent, Dailymotion, DRM, EXTINF, EXTM, ffmpeg, HDS, hls, HLS, jbernard, jpg, js, JS, lang, MPD, MPEG, MSS, multi, Nginx, playlist, png, QuickTime, tv, workflow, Youtube

**Comment by ElevenWilson** [comment]
30 words might be misspelled, please check them: API, Appshot, baseurl, Bitmovin, codec, ContentComponent, Dailymotion, DRM, EXTINF, EXTM, ffmpeg, HDS, hls, HLS, jbernard, jpg, js, JS, lang, MPD, MPEG, MSS, multi, Nginx, playlist, png, QuickTime, tv, workflow, Youtube

**Comment by ElevenWilson** [comment]
30 words might be misspelled, please check them: API, Appshot, baseurl, Bitmovin, codec, ContentComponent, Dailymotion, DRM, EXTINF, EXTM, ffmpeg, HDS, hls, HLS, jbernard, jpg, js, JS, lang, MPD, MPEG, MSS, multi, Nginx, playlist, png, QuickTime, tv, workflow, Youtube

**Comment by ElevenWilson** [comment]
30 words might be misspelled, please check them: API, Appshot, baseurl, Bitmovin, codec, ContentComponent, Dailymotion, DRM, EXTINF, EXTM, ffmpeg, HDS, hls, HLS, jbernard, jpg, js, JS, lang, MPD, MPEG, MSS, multi, Nginx, playlist, png, QuickTime, tv, workflow, Youtube

File: _posts/en/2017-07-19-video-live-dash-hls.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "DASH & HLS : The two main video stream protocols"
+excerpt: DASH & HLS, two protocols to stream them all.
+authors:
+    - jbernard
+lang: en
+permalink: /en/video-live-dash-hls/
```
Comment (VEBERArnaud) [inline _posts/en/2017-07-19-video-live-dash-hls.md:8]:
```suggestion
permalink: /video-live-dash-hls/
```

`/en` is automatically prepended to the permalink for `_posts/en/` posts

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Review by ElevenTom — APPROVED** [review]

**Pull request merged**